db/app.js

```diff
@@ -0,0 +1,26 @@
+const express = require("express");
+const { handlePsqErrors, handleCustomErrors, handleServerErrors } = require("./errors");
+const { getAllTopics, getAllEndpoints, getArticleByArticleID } = require("./app.controller");
+// const apiRouter = require('./routes/api')
+
+const app =express()
+
+app.use(express.json())
+
+app.get('/api/topics', getAllTopics)
+app.get('/api/articles/:article_id',getArticleByArticleID)
+app.get("/api",getAllEndpoints)
+
+
+
+app.all('*' ,(req,res,next) =>{
+    res.status(404).send({msg:"Not Found"})
+});
```

Comment:
Good inclusion of a custom error handler for unhandled endpoints. Is there any testing for this though?